docs/layers.md

```diff
@@ -2140,9 +2140,12 @@ This **optional** layer is meant to be updated every few minutes, due to the hig
 
 #### `optional` properties:
 
-The following properties should only be included when **roads** layer features doesn't include `linear_ref_id`: identifier link to a linear references system, eg [SharedStreet](https://github.com/sharedstreets/sharedstreets-ref-system/blob/master/OSMLR.md). In that case the traffic data should be delivered without geometry and run-time linked with the **roads** layer. But baring that, the following `optional` properties enable sizing and layering of the traffic overlay with respect to the roads network:
+If the roads layer features include linear_ref_id, then traffic flow layer features may include the following, in which case the traffic data should be delivered without geometry and run-time linked with the roads layer.
```

Comment:
Just a nit-pick, I think it would be clearer that we're referring to a property name if `linear_ref_id` were marked-up with backticks. It seems to have emerged as a convention, although probably isn't 100% observed everywhere in this document.

Reply:
Lucky you mentioned, I needed to update the Incidents with the separated paragraphs too